src/Hl7.Fhir.Serialization.R4B.Tests/SerializationExcexptionHandlersJson.cs

```diff
@@ -0,0 +1,312 @@
+﻿using Hl7.Fhir.Model;
```

Comment:
I think these test classes can go into the Shared test project? Why would we only want them from R4B?

Reply:
Was only there as that's the version I was working in (and didn't have to stress over properties across versions giving different results)